api/rest/worker_node.go

```diff
@@ -23,12 +23,14 @@ import (
 
 	"github.com/caicloud/cyclone/api"
 	"github.com/caicloud/cyclone/pkg/log"
+	"github.com/caicloud/cyclone/pkg/osutil"
 	"github.com/caicloud/cyclone/store"
 	"github.com/emicklei/go-restful"
 	docker_client "github.com/fsouza/go-dockerclient"
 )
 
-const WORKER_NODE_DOCKER_VERSION = "1.10.1"
+// WORKER_NODE_DOCKER_VERSION defines the required docker version in worker node.
+const WORKER_NODE_DOCKER_VERSION = "WORKER_NODE_DOCKER_VERSION"
```

Comment:
I changed here because travis's docker version is 1.12.0, and this version could pass e2e tests.